nfpm.yaml

```diff
@@ -0,0 +1,56 @@
+name: pekwm
+arch: amd64
+platform: linux
+
+version: v${SEMVER}
+
+version_schema: semver
+
+section: x11
+priority: optional
+
+maintainer: Claes Nästén <pekdon@gmail.com>
+description: |
+  very light window manager
+  . 
+  Pekwm is a window manager that once up on a time was based on the aewm++ window
+  manager, but it has evolved enough that it no longer resembles aewm++ at all.
+  It has a much expanded feature-set, including window grouping (similar to ion,
+  pwm, or fluxbox), autoproperties, xinerama, keygrabber that supports keychains,
+  and much more.
+  .
+  Some of its characteristics are:
+   * Lightweight and Unobtrusive, a window manager shouldn't be noticed.
+   * Very configurable, everyone work and think in different ways.
+   * Automatic properties, for all the lazy people, make things appear as they
+     should when starting applications.
+   * Chainable Keygrabber, usability for everyone.
+
+homepage: https://github.com/pekdon/pekwm/
```

Comment:
This is actually https://www.pekwm.se , and the GitHub address https://github.com/pekwm/pekwm